### PR TITLE
EVEREST-762 Added support for single storage only

### PR DESCRIPTION
**Pull request by filipmikes1**
https://perconadev.atlassian.net/browse/EVEREST-762

### CODE REVIEW

File: apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx [hunk 1]
```diff
@@ -32,19 +32,27 @@ export const ScheduledBackupModalForm = () => {
...
+      setValue(ScheduleFormFields.storageLocation, {
+        name: dbClusterActiveStorage,
+      });
+    }
+  }, [dbClusterActiveStorage]);
+
   return (
     <ScheduleForm
       allowScheduleSelection={mode === 'edit'}
+      disableStorageSelection={!!dbClusterActiveStorage}
```
Comment (fabio-silva) [inline apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx:55]: shouldn't this be only disabled for MongoDB?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx [hunk 2]
```diff
@@ -32,19 +32,27 @@ export const ScheduledBackupModalForm = () => {
...
-
   useEffect(() => {
     if (mode === 'edit' && setSelectedScheduleName) {
       setSelectedScheduleName(scheduleName);
     }
   }, [scheduleName, mode, setSelectedScheduleName]);
 
+  useEffect(() => {
+    if (dbClusterActiveStorage) {
+      setValue(ScheduleFormFields.storageLocation, {
```
Comment (solovevayaroslavna) [inline apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx:46]: don't we need to trigger validation in this place?

File: apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx (hunk 1 above)
Comment (filipmikes1) [reply apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx:55]: Yes, but dbClusterActiveStorage is returned from backend only for mongo so I think that it is unnecessary

File: apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx (hunk 2 above)
Comment (filipmikes1) [reply apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx:46]: You mean to check if dbClusterActiveStorage is correct? It is a name that we already selected before and backend is just returning it so I don't think it is necessary

Comment (fabio-silva) [reply apps/everest/src/pages/db-cluster-details/backups/scheduled-backup-modal/scheduled-backup-modal-form/scheduled-backup-modal-form.tsx:46]: 👍

**Review by fabio-silva — APPROVED** [review]

**Review by solovevayaroslavna — APPROVED** [review]

**Pull request merged**